### ISSUE TITLE
Ensure content is not too difficult to understand

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#55]
Most content should be written so people with a lower secondary level of education can understand it. Where this is not possible, supplemental content or an alternative version should be provided.

This check highlights pages with a reading age of 16 years or higher. Only pages with at least 100 words of content (excluding navigation, footers, etc) are considered.

For more information, see [WCAG SC 3.1.5](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-supplements.html).
